dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/Constants.java

```diff
@@ -852,4 +877,100 @@ private Constants() {
     public static final String TYPE_MACHINE_LEARNING = "MachineLearning";
 
     public static final String DATASOUCE = "datasource";
+
+    /**
+     * spi constants
+     */
+    /** alert plugin param field string **/
+    public static final String STRING_PLUGIN_PARAM_FIELD = "field";
+    /** alert plugin param name string **/
+    public static final String STRING_PLUGIN_PARAM_NAME = "name";
+    /** alert plugin param props string **/
+    public static final String STRING_PLUGIN_PARAM_PROPS = "props";
+    /** alert plugin param type string **/
+    public static final String STRING_PLUGIN_PARAM_TYPE = "type";
+    /** alert plugin param title string **/
+    public static final String STRING_PLUGIN_PARAM_TITLE = "title";
+    /** alert plugin param value string **/
+    public static final String STRING_PLUGIN_PARAM_VALUE = "value";
+    /** alert plugin param validate string **/
+    public static final String STRING_PLUGIN_PARAM_VALIDATE = "validate";
+    /** alert plugin param options string **/
+    public static final String STRING_PLUGIN_PARAM_OPTIONS = "options";
+    /**plugin param emit string **/
+    public static final String STRING_PLUGIN_PARAM_EMIT = "emit";
+
+    /** string true */
+    public static final String STRING_TRUE = "true";
+    /** string false */
+    public static final String STRING_FALSE = "false";
+    /** string yes */
+    public static final String STRING_YES = "YES";
+    /** string no */
+    public static final String STRING_NO = "NO";
+
+    public static final String SMALL = "small";
+
+    public static final String CHANGE = "change";
+
+    public static final String SPRING_DATASOURCE_MIN_IDLE = "spring.datasource.minIdle";
+
+    public static final String SPRING_DATASOURCE_MAX_ACTIVE = "spring.datasource.maxActive";
+
+    public static final String SPRING_DATASOURCE_TEST_ON_BORROW = "spring.datasource.testOnBorrow";
+
+    /**
+     *  support hive datasource in one session
+     */
+    public static final String SUPPORT_HIVE_ONE_SESSION = "support.hive.oneSession";
+
+    /**
+     * driver
+     */
+    public static final String ORG_POSTGRESQL_DRIVER = "org.postgresql.Driver";
```

Comment:
Can we split this to `DatasourceConstants`? this class is huge now.

Reply:
done